app/models/Level.ozar.coffee

```diff
@@ -69,7 +71,7 @@ module.exports = class Level extends CocoModel
 
   denormalize: (supermodel, session, otherSession) ->
     o = $.extend true, {}, @attributes
-    if o.thangs and @isType('hero', 'hero-ladder', 'hero-coop', 'course', 'course-ladder', 'game-dev', 'web-dev')
+    if o.thangs
```

Comment:
Because it was removed from `.coco` by this commit:
https://github.com/codecombat/codecombat/commit/cf711044875634f5fcb20ff4e5339135901df63b#diff-c34af3f71494bbc8e258e0669fcdf5fa24dd5594968fde0b4729f3ea24c144faL71-R73
So I thought it could go to ozar as well.